swift-sdk/Internal/AnonymousUserManagerProtocol.swift

```diff
@@ -13,5 +13,5 @@ import Foundation
     func trackAnonUpdateUser(_ dataFields: [AnyHashable: Any])
     func updateAnonSession()
     func getAnonCriteria()
-    func syncNonSyncedEvents()
+    func syncEvents()
```

Comment:
Question on why are we deciding to stop using the `syncNonSyncedEvents` method that just adds a delay in execution of the `syncEvents` method?

On the InternalIterableAPI method `attemptAndProcessMerge` we can easily still call `syncNonSyncedEvents` instead right? Is there any reason why we want to use the `syncEvents` method directly?

Reply:
Calling that function was clearing the local storage for some odd reason

Reply:
Hm, I see. Maybe there's a race condition issue then?